airbyte-integrations/connectors/source-retently/source_retently/schemas/customers.json

```diff
@@ -1,28 +1,52 @@
 {
   "type": "object",
   "properties": {
-    "id": { "type": "string" },
-    "email": { "type": "string" },
-    "firstName": { "type": "string" },
-    "lastName": { "type": "string" },
-    "companyName": { "type": "string" },
-    "companyId": { "type": "string" },
+    "id": {
+      "type": "string"
+    },
+    "email": {
+      "type": ["string", "null"]
+    },
+    "firstName": {
+      "type": ["string", "null"]
+    },
+    "lastName": {
+      "type": ["string", "null"]
+    },
+    "companyName": {
+      "type": ["string", "null"]
+    },
     "tags": {
       "type": "array",
-      "items": { "type": "string" }
+      "items": {
+        "type": ["string", "null"]
+      }
+    },
+    "createdDate": {
+      "type": ["string", "null"],
+      "format": "date-time"
     },
-    "createdDate": { "type": "string" },
     "properties": {
       "type": "array",
-      "items": {
-        "type": "object",
-        "properties": {
-          "label": { "type": "string" },
-          "name": { "type": "string" },
-          "type": { "type": "string" },
-          "value": { "type": "string" }
+      "items": [
```

Comment:
@sajarin tests are failing because the `items` object is an array here where it should be an object.

Reply:
Looking into fixing that now!